README.md

```diff
@@ -1 +1,180 @@
-Hello
+<a name="readme-top"></a>
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="readme-assets/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>React-BookStore</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+## 📑 Table of Contents
+- [📊 Math Magician ](#-React-Bookstore-)
+  - [🛠 Built With ](#-built-with-)
+    - [🧰 Tech Stack ](#-tech-stack-)
+    - [💎 Key Features ](#-key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Deployment](#deployment)
+  - [📧 Contact ](#-contact-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [💖 Show your support ](#-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📊 Math Magician <a name="about-project"></a>
+
+A web-based application to understand about the basics of React - Redux 
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### 🧰 Tech Stack <a name="tech-stack"></a>
+
+- ![React](https://img.shields.io/badge/react-%2320232a.svg?style=for-the-badge&logo=react&logoColor=%2361DAFB)
+- ![Redux](https://img.shields.io/badge/redux-%23593d88.svg?style=for-the-badge&logo=redux&logoColor=white)
+- ![React Router](https://img.shields.io/badge/React_Router-CA4245?style=for-the-badge&logo=react-router&logoColor=white)
+- ![Render](https://img.shields.io/badge/Render-%46E3B7.svg?style=for-the-badge&logo=render&logoColor=white)
+
+<!-- Features -->
+
+### 💎 Key Features <a name="key-features"></a>
+
+- Generate a comprehensive Folder Structure for React App 💻
+- Understand the concept of setting up a project with redux 📈
+- Beautifully designed and easy-to-use 🎨
+- Continuous Monitoring 📸
+- Development Tools : Auto refresh and Hot Reload 🔗
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+***Live*** :  [Math Magician](https://63f89b28ed3dd406ccf11729--ahr-React-Bookstore.netlify.app/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] It would be nice if you fix your Live Demo link which is currently broken :fearful: Please fix it to make your readme file descriptive and professional :+1:

Reply:
# Thanks for your wonderful insight. But I think the activity requirement is just to setup the project. 
so Live will be coming soon.

Reply:
Actually, you are supposed to make your readme file descriptive which is a General Requirement of your project and there are checks on your readme file in each milestone so it would be nice if you make your readme file descriptive :+1: